src/main.rs

```diff
@@ -1,3 +1,79 @@
+use clap::Parser;
+use clap::Subcommand;
+
+#[derive(Subcommand)]
+enum SequencerCommands {
+    /// Create a new block and push it on chain
+    Push,
+
+    /// Pull new accounts from the contract
+    Pull,
+}
+
+#[derive(Subcommand)]
+enum UserCommands {
+    /// Create a new account in the rollup by communicating with the contract
+    Enter,
```

Comment:
For me this PR is good enough for the initial mockup of commands. But just thinking out loud, we would also need some for create a key pair and one to sign transactions as an user.

Reply:
Yeah, we'll add them when we'll be implementing those features. Right now there's not enough info to know what options we'll need. My goal here was just to get some initial structure working and compiling, that we can later adjust to our needs.